cloud/README.md

```diff
@@ -0,0 +1,65 @@
+# Please do not publish a solution in either a fork or clone. Keep all of your work local!
+#### If you do want to store your work, we ask that you keep it in a private repo (github lets everyone do this now).
+
+## Introduction
+
+This project is designed to get you familiar with some basic functionalities of our ArchFX Cloud product.
+
+Arch has built a complete framework to help factories monitoring their production lines. Arch FX Cloud is the core of
+this product, as it is the main user interface but also the link with all our other products.
+
+It is basically a webapp served by a Django server, with all the needed backend tools.
+
+To test the skills you will need to be part of the Cloud team, we will build a small API running with Django.
+
+*It should take less than 1 hour to complete. If you think some parts need more, please tell us.*
+
+## Description
+
+The goal of this exercise is to build an API that will allow a user to register time blocks with a name
+and then query the resulting timeline.
+
+1. Setup a Django server within a Docker container. This server will need a PostgreSQL database to store models, which
+   will also be into a Docker container. We recommend describing these containers using docker-compose.
+2. Define the `Timeblock` model and migrations to save it in DB.
+   - id: int (auto-increment)
+   - name: str
+   - start_time: datetime
+   - end_time: datetime
+3. Add API routes using the `django-rest-framework` library. You have to choose the names and methods.
```

Comment:
And here, I think it doesn't tell the candidate too much to show a skeleton of where you want them to put their routes. They will still need to modify all of the right files (views, urls, etc).